app/helpers/icons_helper.rb

```diff
@@ -114,6 +114,10 @@ def chat_message_icon
     fa_icon('commenting-o')
   end
 
+  def comment_icon
+    fa_icon('comment')
```

Comment:
I want to keep all icons as encapsulated methods for two reasons:
- We might want to use another icon as our "comment" icon, e.g. 'comment-o'.
- I usually disable buttons with `data-disable-with` => "#{spinner_icon} Text" and I had problems to make that work any other way.